docs/smoothing.rst

```diff
@@ -27,6 +27,22 @@ single-resolution :class:`~spectral_cube.SpectralCube` instances, but for a
 :class:`~spectral_cube.VaryingResolutionSpectralCube`, the convolution kernel
 will be different for each slice.
 
+Common Beam selection
+^^^^^^^^^^^^^^^^^^^^^
+You may want to convolve your cube to the smallest beam that is still larger
+than all contained beams.  To do this, you can use the
+:func:`~radio_beam.common_beam.commonbeam` tool.  For example::
+
+    common_beam = cube.beams.common_beam()
+    new_cube = cube.convolve_to(common_beam)
+
+Sometimes, you'll encounter the error "Could not find common beam to deconvolve
+all beams." This is a real issue, as the algorithms we have in hand so far do
+not always converge on a common containing beam.  You can try to change the
+tolerance used in the :func:`~radio_beam.commonbeam.getMinVolEllipse` code by
```

Comment:
The "smaller" part might be confusing. We should recommend increasing the tolerance and/or epsilon to address this error. How about:

You can try to change the tolerance and epsilon used in the :func:`~radio_beam.commonbeam.getMinVolEllipse` code to get the algorithm to converge. Specifying `tolerance` will set the convergence criteria for the algorithm; smaller values will take longer to reach convergence. Try increasing the tolerance from `tolerance=1e-4` to get the algorithm to converge. `epsilon` sets the fraction to overestimate the beam perimeters. Increasing epsilon from the default `epsilon=5e-4` will ensure the algorithm converges at the expense of overestimating the common beam area. The smallest `epsilon` where convergence is reached should be used.

Reply:
I disagree; see below.

But, you can also directly suggest changes now!
![image](https://user-images.githubusercontent.com/143715/48807861-1be8c500-ecdc-11e8-958e-db30dcee1109.png)